ui/package.json

```
@@ -40,7 +40,7 @@
   "scripts": {
     "analyze": "source-map-explorer 'build/static/js/*.js'",
     "start": "react-app-rewired start",
-    "build": "react-app-rewired build",
+    "build": "react-app-rewired --max_old_space_size=3072 build",
```

I'm thinking maybe a child node process is started without this V8 option, let's try to set this via NODE_OPTIONS env.

```suggestion
    "build": "NODE_OPTIONS='--max-old-space-size=3072' react-app-rewired --max_old_space_size=3072 build",
```

---

I checked (in `ps` output while doing a build), it is being passed through as-is:
```
  11533 ?        Ss     0:00          \_ /bin/bash /entrypoint.sh build
  11560 ?        Sl     0:00              \_ npm
  11571 ?        S      0:00                  \_ sh -c react-app-rewired --max_old_space_size=3072 build
  11572 ?        Sl     0:00                      \_ node /home/node/node_modules/.bin/react-app-rewired --max_old_space_size=3072 build
  11579 ?        Rl     0:12                          \_ node --max_old_space_size=3072 /home/node/node_modules/react-app-rewired/scripts/build.js
```